.circleci/config.yml

```diff
@@ -905,12 +892,20 @@ workflows:
               only:
                 - auto
       - docker-publish:
+          context: docker
+          filters:
+            branches:
+              only:
+                #- /^test-[\d|.]+$/
```

Comment:
turned off for testing..... don't want to push a bunch of images to dockerhub for no reason.